addons/gui/functions/fnc_updateInjuryList.sqf

```diff
@@ -213,7 +217,7 @@ switch (GET_FRACTURES(_target) select _selectionN) do {
 };
 
 // Display cyanosis in overview tab, only when head/arms are selected
-if (EGVAR(breathing,showCyanosis) && _selectionN in [0,2,3]) then {
+if (EGVAR(breathing,showCyanosis) && _selectionN in [0,2,3,4,5]) then {
```

Comment:
consider it as basic feedback but i can understand you see cyanosis on fingers or ears/nose but its kinda not visible on legs that are hidden by pants and boots